content/messaging/agent-chat-api/v3.3/index.mdx

```diff
@@ -2197,40 +2197,55 @@ It returns the list of Customers.
-| `filters.chats_count.<range_filter_type>` **\*\*\***                           | `object`  |                           |
-| `filters.threads_count.<range_filter_type>` **\*\*\***                         | `object`  |                           |
-| `filters.visits_count.<range_filter_type>` **\*\*\***                          | `object`  |                           |
-| `filters.created_at.<date_range_filter_type>`**\*\*\*\***                      | `object`  |                           |
-| `filters.agent_last_event_created_at.<date_range_filter_type>` **\*\*\*\***    | `object`  |                           |
-| `filters.customer_last_event_created_at.<date_range_filter_type>` **\*\*\*\*** | `object`  |                           |
+| Parameter                                                                        | Data type | Notes                     |
+|----------------------------------------------------------------------------------|-----------|---------------------------|
+| `page_id`                                                                        | `string`  |                           |
+| `limit`                                                                          | `number`  | Default: 10, maximum: 100 |
+| `sort_order` **\***                                                              | `string`  | Default: `desc`           |
+| `sort_by` **\*\***                                                               | `string`  | Default: `created_at`     |
+| `filters`                                                                        | `object`  |                           |
+| `filters.country.<string_filter_type>` **\*\*\***                                | `object`  |                           |
+| `filters.email.<string_filter_type>` **\*\*\***                                  | `object`  |                           |
+| `filters.name.<string_filter_type>` **\*\*\***                                   | `object`  |                           |
+| `filters.customer_id.<string_filter_type>` **\*\*\***                            | `object`  |                           |
+| `filters.chats_count.<range_filter_type>` **\*\*\*\***                           | `object`  |                           |
+| `filters.threads_count.<range_filter_type>` **\*\*\*\***                         | `object`  |                           |
+| `filters.visits_count.<range_filter_type>` **\*\*\*\***                          | `object`  |                           |
+| `filters.created_at.<date_range_filter_type>`**\*\*\*\*\***                      | `object`  |                           |
+| `filters.agent_last_event_created_at.<date_range_filter_type>` **\*\*\*\*\***    | `object`  |                           |
+| `filters.customer_last_event_created_at.<date_range_filter_type>` **\*\*\*\*\*** | `object`  |                           |
 
 **\*)**
 
 `sort_order` can take the following values:
 
-- `asc` - oldest customers first
```

Comment:
Why did we get rid of those notes "oldest/newest customers first"?

Reply:
Because previously we sorted results by creation time always so when sorting asc then oldest customers were first and when using desc newest customers were first. When we added other sorting options it is no longer valid because when sorting descending by let say threads_count then customers with most threads are first

Reply:
true :) thanks